utils/_context/_scenarios.py

```diff
@@ -844,6 +840,13 @@ class scenarios:
         },
     )
 
+    appsec_api_security = EndToEndScenario(
```

Comment:
Could you add a comment explaining what this scenario do ?

Reply:
This is a scenario for the new feature/product **_API Security_**. I thought the name was explicit enough.